plugins/pencilblue/controllers/admin/sites/site_form.js

```diff
@@ -16,7 +16,6 @@
  */
```

Comment:
Hey @brianhyder, this file is 2-space tabbed rather than the conventional 4. I left it 2-space tabbed for now so the PR wasn't spammy. Shall I convert it to 4-space tabs before we're done with this PR?